ui/app/reducers/metamask.js

```diff
@@ -24,6 +24,12 @@ function reduceMetamask (state, action) {
     send: {
```

Comment:
I think this is a good time to introduce nested reducers here. Not urgent now. Let's make a task for refactoring